src/helpers/get-public-env.ts

```diff
@@ -1,7 +1,13 @@
+import * as log from '../output/log';
+
 /**
  * Gets a list of environment variables that start with `NEXT_PUBLIC_`.
  */
 export function getPublicEnv() {
+  log.info(
+    `Get environment variables prefixed with 'NEXT_PUBLIC_' from process.env`
+  );
```

Comment:
Can we add a test for this line?

Next.js uses past tense for their messages, so maybe: `Read environment variables prefixed with 'NEXT_PUBLIC_' from process.env.`

Also, should we add it after we actually read it?